src/zkevm_specs/evm/typing.py

```diff
@@ -4,6 +4,34 @@
 from .table import TxContextFieldTag
 
 
+class Block:
+    coinbase: U160
+    gas_limit: U64
+    block_number: U256
+    time: U256
+    difficulty: U256
+    base_fee: U256
+    previous_256_block_hashes: Sequence[U256]
+
+    def __init__(
+        self,
+        coinbase: U160 = 0x10,
```

Comment:
any reference for the default coinbase address?

Reply:
currently block fields are filled into block context table, such table will be as pub input eventually.

Reply:
block fields will be public input in the future , so just pass them for now .